docs/UsersGuide/source/BuildRunSRW.rst

```diff
@@ -0,0 +1,847 @@
+
+.. attention::
+
+   All UFS applications support `four platform levels <https://github.com/ufs-community/ufs-srweather-app/wiki/Supported-Platforms-and-Compilers>`_. The steps described in this chapter will work most smoothly on preconfigured (Level 1) systems. On Level 1 systems, all of the required libraries for building community releases of UFS models and applications are available in a central location. This guide can serve as a starting point for running the SRW App on other systems, too, but the user may need to perform additional troubleshooting. 
+
+.. note::
+   The :ref:`container approach <QuickstartC>` is recommended for a smoother build and run experience. Building without a container allows for the use of the Rocoto workflow manager and may allow for more customization. However, the non-container approach requires more in-depth troubleshooting skills, especially on Level 3 and 4 systems, and is less appropriate for beginners. 
+
+The overall procedure for generating an experiment is shown in :numref:`Figure %s <AppOverallProc>`, with the scripts to generate and run the workflow shown in red. The steps are as follows:
+
+   * :ref:`Install prerequisites <HPCstackInfo>`
+   * :ref:`Clone the SRW App from GitHub <DownloadSRWApp>`
+   * :ref:`Check out the external repositories <CheckoutExternals>`
+   * :ref:`Set up the build environment <SetUpBuild>`
+   * :ref:`Build the executables <BuildExecutables>`
+   * :ref:`Download and stage data <Data>`
+   * :ref:`Optional: Configure a new grid <GridSpecificConfig>`
+   * :ref:`Generate a regional workflow experiment <GenerateForecast>`
+      * :ref:`Configure the experiment parameters <UserSpecificConfig>`
+      * :ref:`Load the python environment for the regional workflow <SetUpPythonEnv>`
+   * :ref:`Run the regional workflow <RocotoRun>` 
+   * :ref:`Optional: Plot the output <PlotOutput>`
+
+.. _AppOverallProc:
+
+.. figure:: _static/FV3LAM_wflow_overall.png
+
+    *Overall layout of the SRW App Workflow*
+
+
+.. _HPCstackInfo:
+
+Install the HPC-Stack
+========================
+
+.. Attention::
+   Skip the HPC-Stack installation if working on a `Level 1 system <https://github.com/ufs-community/ufs-srweather-app/wiki/Supported-Platforms-and-Compilers>`_ (e.g., Cheyenne, Hera, Orion, NOAA Cloud).
```

Comment:
I can, but an "e.g.," list is a set of examples and doesn't have to be all-inclusive ("e.g." means "exempli gratia", or "for example," in English). That's what the link is for! ;) Otherwise, shouldn't I be adding Gaea and WCOSS machines, too? Another option is to replace "e.g." with "for example," if you think that's clearer.

Reply:
Totally fine as is. You are right that it is correct with e.g. If you add all I think it should be changed to i.e. No problem as is.